src/blocks/disk_space.rs

```diff
@@ -159,7 +166,10 @@ impl Block for DiskSpace {
         let statvfs = Statvfs::for_path(Path::new(self.path.as_str()))
             .block_error("disk_space", "failed to retrieve statvfs")?;
         let result;
-        let converted;
+        let mut converted = 0.0f64;
+		let mut converted_str = String::new();
```

Comment:
Looks like there are some whitespace issues here. Can you stage `rustfmt`-ified versions of the regions you've modified?